### PR TITLE
[POA-2003] Update observability-ir version

### PR DESCRIPTION
**Pull request by mudit-postman**
* Updated observability-ir version
* Updated `generated-types.go` with the new changes

### CODE REVIEW

**Review by mgritter — APPROVED** [review]
Isn't this blocked until the IR changes are merged so you can refer to the trunk commit?

Anyway, looks fine as long as this is the correct akita-ir version.

**Comment by liujed** [comment]
> Isn't this blocked until the IR changes are merged so you can refer to the trunk commit?

Yeah, we should add a CI check for this.

**Review by liujed — APPROVED** [review]

**Pull request merged**
